Consul/Properties/AssemblyInfo.cs

```diff
@@ -1,18 +1,5 @@
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Consul.NET")]
```

Comment:
The `Title` and `Product` elements will change to `"Consul"` instead of `"Consul.NET"`. Maybe that's fine?

Reply:
True. I guess it's fine... or if it turns out to be a breaking change for a user, that user is definitly doing something shady :D Happy to explicitly set it to the old Title if it makes sense.

Reply:
Added it back